com.unity.render-pipelines.universal/Shaders/Nature/SpeedTree8.shader

```diff
@@ -66,7 +66,7 @@ Shader "Universal Render Pipeline/Nature/SpeedTree8"
             #pragma shader_feature_local _WINDQUALITY_NONE _WINDQUALITY_FASTEST _WINDQUALITY_FAST _WINDQUALITY_BETTER _WINDQUALITY_BEST _WINDQUALITY_PALM
             #pragma shader_feature_local EFFECT_BILLBOARD
             #pragma shader_feature_local EFFECT_HUE_VARIATION
-            //#pragma shader_feature_local EFFECT_SUBSURFACE // GI dependent.
+            #pragma shader_feature_local EFFECT_SUBSURFACE
```

Comment:
Could this be made `shader_feature_local_fragment` ?

Reply:
Changed in d2a779d

Reply:
After testing on yamato, changing to shader_feature_local_fragment breaks iphone tests. Reverting.